ToDo.txt

```diff
@@ -0,0 +1,2 @@
+To Do: 
```

Comment:
redundant file? Also, I'd add things like `# TODO:` directly in the source code. Most editors have highlighting for that and it's more easily searchable.